centreon/plugins/script_sql.pm

```diff
@@ -48,7 +48,7 @@ sub new {
                                   );
     $self->{version} = '1.0';
     %{$self->{modes}} = ();
-    %{$self->{sql_modes}} = ('dbi' => 'centreon::plugins::dbi');
+    $self->{sql_modes}{api} = 'centreon::plugins::dbi';
```

Comment:
cannot imagine that this change doesn't break things, did you really test this ? that is totally different

Reply:
yes there are 2 different `(custom_)?modes` pattern caught and I wrongly rewrote this line using the wrong pattern...
I mainly concentrated to include every needed file, and did some basic testing with a few plugins to be sure they launched correctly.
SQL ones do now.
Thank you for your feedback :+1:

Reply:
@CPbN please make it looks as it was originally as a one line :) thanks

Reply:
It will complicate pattern matching, but let's have a look !

Reply:
Done, I've found a way without complicating pattern matching :+1: